src/main/java/se/sundsvall/supportmanagement/integration/db/util/NamespaceShortCode.java

```diff
@@ -0,0 +1,22 @@
+package se.sundsvall.supportmanagement.integration.db.util;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public enum NamespaceShortCode {
+
+	KC("CONTACTCENTER");
+
+	private final String namespace;
+
+	NamespaceShortCode(final String namespace) {
+		this.namespace = namespace;
+	}
+
+	public static NamespaceShortCode findByNamespace(final String namespace) {
+		return Arrays.stream(values())
+			.filter(namespaceShortCode -> namespaceShortCode.namespace.equals(namespace))
+			.findFirst()
+			.orElseThrow(() -> new NoSuchElementException("No namespace shortcode found for " + namespace));
```

Comment:
Då vi inte har kontroll över vilka namespaces som klienter kommer använda så är förslaget att returnera hela namespace:t i det fall ingen förkortning för det hittas. På så sätt skapar vi inget beroende att tjänsten måste uppdateras för varje nytt namespace (läs klient) som vil använda den. Det som kan hända är att prefixet före ärendenumret blir långt, men det borde vara rätt enkel sak att uppdatera i samband med att namespace:t läggs till här när/om klienten önskar.

Ett annat förslag är att istället för att ha en enum, helt enkelt ta första och sista bokstaven i namespace:t i upper-case, vilket för contactcenter skulle rendera i CR-<datum><nr>. Vet inte ifall det krävs att ärendet prefixas med precis KC, men en sådan lösning skulle inte behöva koduppdatering för varje nytt namespace.

Vad tror du om det? Värt att kolla med Jakob om kanske?

Reply:
Frågan är ställd till jakob som skulle kolla med intressenter och återkopplar. To be continuted

Reply:
Uppdatering: Shortcode behöver vara ett mappat värde ur ett användarvänlighetsperspektiv. Det är att föredra att utifrån ärendenummer kunna utläsa vart det hör hemma. Tjänsten kommer att behöva uppdateras inför varje ny namespace som vill använda den, att lägga till fullständigt namespace är inte önskvärt. 

Kastar vi fel kommer (förhoppningsvis) anslutande namespace upptäcka att de behöver se till att det finns en shortcode när de börjar testa mot SM.